### DISCUSSION TITLE
Encoder volume control!

### DISCUSSION THREAD

**Discussion by Nasty76** [tttapa/Control-Surface#611] [General]
maybe i  m blind but i try to arduino uno + hairless midi , to control my voicemeter potato but i dont undertand how i can use encoder as strip 1 volume etc. Thanks!

**Comment by tttapa** [comment]
Either use the `CCRotaryEncoder` or the `CCAbsoluteEncoder` class, and then use the MIDI learn functionality explained here: https://voicemeeter.com/volume-control-how-to-change-the-volume-of-your-main-output-without-the-mouse/

I'm not sure if the plugin you're using supports relative CC messages.
